profiles/default/linux/make.defaults

```diff
@@ -11,7 +11,7 @@
 
 
 # Default starting set of USE flags for all default/linux profiles.
-USE="crypt ipv6 ncurses nls pam readline ssl tcpd zlib"
+USE="bzip2 crypt ipv6 lzma ncurses nls pam readline ssl tcpd zlib zstd"
```

Comment:
I think I'm overall happy with this but I'd like to, at some point, audit the packages with these 3 new flags and figure out if it makes sense to have them and +IUSE them if necessary. Then drop it again.

Maybe we should put a TODO above it and/or a note to consider flipping IUSE defaults instead for future changes and link to the thread for some (historical) context.

Reply:
I'm also a little bit interested in whether you can actually get a built system (like with --root) without these, but it doesn't really affect the decision making for this change.

Reply:
Agreed, `optfeature` would be even better.

I think it's impossible to have a Gentoo system without `sys-libs/zlib`. It's an unconditional dependency of `sys-devel/binutils`, `sys-devel/gcc`, and `sys-devel/llvm`.

Reply:
> I think it's impossible to have a Gentoo system without `sys-libs/zlib`. It's an unconditional dependency of `sys-devel/binutils`, `sys-devel/gcc`, and `sys-devel/llvm`.

So enable it in profiles, not in hundreds of ebuilds. This isn't package specific, so it doesn't belong in IUSE defaults.